ethereumetl/streaming/postgres_tables.py

```diff
@@ -105,6 +105,21 @@
     Column('block_hash', String),
 )
 
+TOKEN_TRANSFERS_V2 = Table(
```

Comment:
`postgres` export is probably not going to be used but just added for consistency in case we use it in the future